### ISSUE TITLE
Some accessibility issues

### ISSUE THREAD

**Issue by NVCDevelopmentTeam** [freeCodeCamp/LearnToCodeRPG#107]
I'm having a hard time reading the question in learn to code rpg and it seems that my screen reader can't read the question body. Also, the inability to change the reader and voice makes it very difficult for me to interact with it. Not to mention, I can't use learn to code rpg with a screen reader like nvda but have to depend on a windows synthesizer like sapi 5. This makes me feel quite slow because I can't speed up voice or read questions actively and use nvda to read questions directly instead of just reading with sapi 5. I think I need to choose the speed, the pitch, and the volume for the set. Also, I want to use nvda to play this game and not necessarily use sapi 5 while it would be annoying or feel weird to play. I hope, there is some solution to this problem. Hopefully, it will work fine with NVDa.

**Comment by NVCDevelopmentTeam** [comment]
The lack of an option to increase or decrease the speed or voice selection for the game is flawed. Also, the conversion from sapi5 to nvda is not yet allowed. I think, should change.
